CHANGELOG.md

```diff
@@ -34,6 +34,8 @@ Main (unreleased)
 
 - Register prometheus discovery metrics. (@mattdurham)
 
+- Don't use the optional config parameter (instance) as the Identity on apache_http integration v2. Resolves #2017 for the apache_http integration. (@rgeyer)
```

Comment:
This feels like it needs some rewording. Can we change this to what it means for the user? Something like:

> The integrations-next apache_http integration will no longer panic if the instance field is not specified. (@rgeyer)

(You also don't have to link to the issue)